lib/evm_database.rb

```diff
@@ -16,6 +16,12 @@ class EvmDatabase
     MiqReport
     RssFeed
     MiqWidget
+    MiqAction
+    MiqEventDefinition
+    Condition
+    MiqPolicy
+    MiqPolicyContent
+    MiqPolicySet
```

Comment:
Submitted ORDERED_CLASSES #8013.

Another way I saw before in some products would be
```ruby
class Foo
  def self.seed
    # Seeding code for Foo
  end

  def self.seed_dependencies
    [Bar]
  end
end
```
Then it is very easy to do topological sort and detect circles: at each stage if there is a class whose all dependencies are met seed it and remove it from dependency lists. If there is not there is surely a circle.